### PR TITLE
noship Microsoft.AspNetCore.Server.IIS

### PR DESCRIPTION
**Pull request by jkotalik**
@JunTaoLuo will this remove Microsoft.AspNetCore.Server.IIS from the meta packages?

### CODE REVIEW

File: build/artifacts.props [hunk 1]
```diff
@@ -147,7 +147,7 @@
     <PackageArtifact Include="Microsoft.AspNetCore.Routing.Abstractions" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
     <PackageArtifact Include="Microsoft.AspNetCore.Routing.DecisionTree.Sources" Category="noship" />
     <PackageArtifact Include="Microsoft.AspNetCore.Server.HttpSys" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
-    <PackageArtifact Include="Microsoft.AspNetCore.Server.IIS" Category="ship" WinAppMetapackage="true" WinAllMetapackage="true" />
+    <PackageArtifact Include="Microsoft.AspNetCore.Server.IIS" Category="noship" WinAppMetapackage="true" WinAllMetapackage="true" />
```
Comment (natemcmaster) [inline build/artifacts.props]: We'll need to cut out more than just this to unship it. You'll need to revert https://github.com/aspnet/Universe/pull/989 too.

Comment (jkotalik) [reply build/artifacts.props]: Hey, let's think about the positives. There will be no native dependencies in the meta packages now!

**Comment by JunTaoLuo** [comment]
What's going on with this? Are we pushing it out to rc or beyond 2.1? If we are just moving it to rc, I'd just mark https://github.com/aspnet/Universe/blob/dev/build/artifacts.props#L312-L313 as noship. If we are committed to not shipping it for 2.1 then revert the change Nate mentioned.

**Comment by Eilon** [comment]
@JunTaoLuo out of 2.1 was my understanding.

**Comment by jkotalik** [comment]
@natemcmaster I added the revert in this PR too.

**Pull request merged**